src/modules/todoList.js

```diff
@@ -0,0 +1,76 @@
+import { descInput } from './elements.js';
+
+class TODO {
+  constructor() {
+    this.index = 0;
+    this.completed = false;
+    this.description = '';
+    this.todos = [];
+  }
+
+  addTodo() {
+    this.description = descInput.value;
+    this.index = this.todos.length;
+    this.todos.push({
+      index: this.index,
+      description: this.description,
+      completed: this.completed,
+    });
+    this.storeTodo();
+  }
+
+  readTodo() {
+    const TODOS = JSON.parse(localStorage.getItem('todos'));
+    if (TODOS) {
+      this.todos = TODOS;
+    } else {
+      this.todos = [];
+    }
+    return this.todos;
+  }
+
+  clearCompleted() {
+    this.todos = this.todos.filter((todo) => todo.completed !== true);
+    this.arrangeIndex();
+    this.storeTodo();
+  }
```

Comment:
https://www.awesomescreenshot.com/video/7517856?key=838c396a0da06127f5641b4ac6048a6e
-[ ] There is a problem with the functionality here it's a simple issue but important when refreshing the page fo the users

Reply:
Thank you @omar25ahmed ,

I really appreciate you point out the lacking feature. I fixed this feature in this iteration.